src/components/styles/BookForm.css

```diff
@@ -1,36 +1,52 @@
-/* BookForm container */
 .book-form {
+  width: 100%;
   display: flex;
-  flex-direction: column;
-  max-width: 400px;
-  margin: 0 auto;
+  justify-content: center;
+  padding: 20px;
+  margin-bottom: 20px;
 }
```

Comment:
- [ ] I see above the form a horizontal line is missing and also `Add New Book` is missing there so Please implement it so it look like the original design.
- [ ] Note: You can use . [Login to Zeplin](https://app.zeplin.io/login). providing the password and email that microverse provided. From there you can use the same color, font, font family and so on. 
### Your design 
![his](https://github.com/Xghayor/Book-store/assets/81727988/3c4920f7-c9ac-4abe-85f2-33e3aa28ed56)

### Original design
![original](https://github.com/Xghayor/Book-store/assets/81727988/e6b5ea39-60cf-41fb-9b9e-0ee298e812ef)